test/probes/isUnsafeCommand.spec.js

```diff
@@ -0,0 +1,130 @@
+// Import Node.js Dependencies
+import { test } from "node:test";
+import assert from "node:assert";
+
+// Import Internal Dependencies
+import { parseScript, getSastAnalysis } from "../utils/index.js";
+import isUnsafeCommand from "../../src/probes/isUnsafeCommand.js";
+
+// CONSTANTS
+const kWarningUnsafeCommand = "unsafe-command";
+
+// Spawn
+
+test("should detect csrutil spawn command", () => {
+  const str = `
+    const { spawn } = require("child_process");
+    spawn("csrutil", ["status"]);
+  `;
+
+  const ast = parseScript(str);
+  const sastAnalysis = getSastAnalysis(str, isUnsafeCommand)
+    .execute(ast.body);
+
+  const result = sastAnalysis.getWarning(kWarningUnsafeCommand);
+  assert.equal(result.kind, kWarningUnsafeCommand);
+  assert.equal(result.value, "csrutil");
```

Comment:
Now I realised that we should output more info for the spawn case, like rebuilding the whole command with args.

Reply:
Otherwise you have more info for exec than spawn